pynest/examples/Potjans_2014/readme.md

```diff
@@ -0,0 +1,35 @@
+# Microcircuit Readme
+
+Authors: Hendrik Rothe, Hannah Bos, Sacha van Albada   
+May 2016
+
+## Description ##
+This is a PyNEST implementation of the microcircuit model by Potjans and Diesmann (2014): The cell-type specific
+cortical microcircuit: relating structure and activity in a full-scale spiking
+network model. Cerebral Cortex: doi:10.1093/cercor/bhs358  
+The original sli version can be found [here](https://github.com/nest/nest-simulator/tree/master/examples/nest/Potjans_2014).
+
+* Files:  
+	* `helpers.py`  
+	Helper functions for the simulation and evaluation of the microcircuit.	
+	* `network.py`  
+	File which gathers all parameters and connects the different nodes with each other.
```

Comment:
Rephrase to: "Gathers..." in acoordance with the descriptions of the other files.